packages/elements/src/components/ino-icon/validate.ts

```diff
@@ -1,54 +1,54 @@
 import { isStr } from './utils';
 
 export const validateContent = (svgContent: string | null) => {
-  if (svgContent) {
-    const div = document.createElement('div');
-    div.innerHTML = svgContent;
+	if (svgContent) {
+		const div = document.createElement('div');
+		div.innerHTML = svgContent;
 
-    // setup this way to ensure it works on our buddy IE
-    for (let i = div.childNodes.length - 1; i >= 0; i--) {
-      if (div.childNodes[i].nodeName.toLowerCase() !== 'svg') {
-        div.removeChild(div.childNodes[i]);
-      }
-    }
+		// setup this way to ensure it works on our buddy IE
+		for (let i = div.childNodes.length - 1; i >= 0; i--) {
+			if (div.childNodes[i].nodeName.toLowerCase() !== 'svg') {
+				div.removeChild(div.childNodes[i]);
+			}
+		}
 
-    // must only have 1 root element
-    const svgElm = div.firstElementChild;
-    if (svgElm && svgElm.nodeName.toLowerCase() === 'svg') {
-      const svgClass = svgElm.getAttribute('class') || '';
-      svgElm.setAttribute('class', (svgClass + ' s-ion-icon').trim());
+		// must only have 1 root element
+		const svgElm = div.firstElementChild;
+		if (svgElm && svgElm.nodeName.toLowerCase() === 'svg') {
+			const svgClass = svgElm.getAttribute('class') || '';
+			svgElm.setAttribute('class', (svgClass + ' s-ion-icon').trim());
 
-      // root element must be an svg
-      // lets double check we've got valid elements
-      // do not allow scripts
-      if (isValid(svgElm as any)) {
-        return div.innerHTML;
-      }
-    }
-  }
-  return '';
+			// root element must be an svg
+			// lets double check we've got valid elements
+			// do not allow scripts
+			if (isValid(svgElm as any)) {
+				return div.innerHTML;
+			}
+		}
+	}
+	return '';
 };
 
 export const isValid = (elm: HTMLElement) => {
-  if (elm.nodeType === 1) {
-    if (elm.nodeName.toLowerCase() === 'script') {
-      return false;
-    }
+	if (elm.nodeType === 1) {
+		if (elm.nodeName.toLowerCase() === 'script') {
+			return false;
+		}
 
-    // tslint:disable-next-line:prefer-for-of
-    for (let i = 0; i < elm.attributes.length; i++) {
-      const val = elm.attributes[i].value;
-      if (isStr(val) && val.toLowerCase().indexOf('on') === 0) {
-        return false;
-      }
-    }
+		// tslint:disable-next-line:prefer-for-of
```

Comment:
Search for 'tslint' and 'tslint.json' files in the entire project and check if occurrences still needed

Reply:
Thank you for your extensive review! I just implemented your feedback - let me know if you have any other suggestions.